test/unittest/xmlparser/XMLProfileParserTests.cpp

```diff
@@ -523,10 +536,118 @@ TEST_F(XMLProfileParserTests, XMLParserSecurity)
 
 #endif
 
+TEST_F(XMLProfileParserTests, file_xml_consumer_append)
+{
+    // First remove previous executions file
+    std::remove("test1.log");
```

Comment:
I think these tests doesn't have to check the behavior of Logger. For this purpose we have its unit tests. These test should check the XML parser do the right thing. Thus, check the consumers or logger configuration was loaded correctly.

Reply:
Changed. Now, these tests only check that the logger is configured correctly.